maps/clarion.dmm

```diff
@@ -29731,7 +29731,7 @@
 /turf/simulated/floor/redblack{
 	dir = 9
 	},
-/area/station/security/main)
+/area/station/security/brig)
```

Comment:
changes to clarion are not permitted at this time

Reply:
Alright, just reverted it then.